src/framework/preferences/css/sass/Enactors.scss

```diff
@@ -2,7 +2,7 @@
  * General prefs editor styling used to apply preferences
  * !important is used for all styles to ensure that Preferences Framework's enactors are able to override the page's default styles.
  */
-@import "utils/Fonts"
+@import "utils/fonts";
```

Comment:
It seems that `@import` is deprecated in favour of `@use`. I think we might as well switch to `@use`.

Reply:
Resolved in 3d193de26. However, the import of Switch can't be replaced with a `@use` statement in its current form.